bin/release

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+# shellcheck source=bin/_config.sh
+source "$(dirname "${BASH_SOURCE[0]}")/_config.sh"
+
+function usage(){
+    echo -e "Usage: ${0} [SITE1] [SITE2] [OPTIONS]
+
+Each site must be the name of a folder in the "sites" directory
+holding a richie based site with customizations. If no sites are
+specified, the script will loop over all sites to make releases.
+
+OPTIONS:
+  -h, --help       print this message
+  -m, --major      force a major release (breaking change)
+  -c, --commit     commit the release to git log
+"
+}
+
+# Bump a site's release according to its changelog.
+#
+# Usage: release SITE IS_MAJOR SHOULD_COMMIT
+#
+#  SITE           string  name of a folder in the "sites" directory
+#  IS_MAJOR       0|1     whether we should force a major release (breaking change)
+#  SHOULD_COMMIT  0|1     whether we should commit the release to the git log
+function release() {
+    site=$1
+    is_major=$2
+    should_commit=$3
+
+    changelog=`sed "/## \[Unreleased/,/## \[/!d" "${SITES_DIRECTORY}/${site}/CHANGELOG.md" | tail -n +2`
+
+    # Extract the last version released from changelog
+    current_version=`echo "$changelog" | grep "## \[" | sed -E "s/.*\[(.*)\].*/\1/"`
+    current_version_array=( ${current_version//./ } )
+
+    # Bump version to above major, minor or revision number depending on changelog content
+    if [[ "${is_major}" == 1  ]]; then
+        # Major release
+        new_version=`echo "$((++current_version_array[0])).0.0"`
+    else
+        if ! echo "${changelog}" | grep "###" &> /dev/null; then
+            # No changes since the last release, do nothing
+            echo -e "${COLOR_INFO}Nothing to release on ${site}${COLOR_RESET}"
+            return 0
+        fi
+        if echo "${changelog}" | grep "###" | grep -v "Fixed" &> /dev/null; then
+            # Minor release
+            new_version=`echo "${current_version_array[0]}.$((++current_version_array[1])).0"`
+        else
+            # Revision release (fix)
+            new_version=`echo "${current_version_array[0]}.${current_version_array[1]}.$((++current_version_array[2]))"`
+        fi
+    fi
+
+    echo -e "${COLOR_INFO}Creating release v${new_version} for ${site}${COLOR_RESET}"
+
+    # Prepare tag for the new releave in the form:
+    # ## [2.10.2]  2020-10-07
+    new_tag=`echo "\\\n## [${new_version}] - $(date +'%Y-%m-%d')"`
+
+    # Update CHANGELOG.md
+    sed -i "s/^\[unreleased\].*$/[unreleased]: https:\/\/github.com\/openfun\/richie-site-factory\/compare\/${site}-${new_version}...HEAD/g" "${SITES_DIRECTORY}/${site}/CHANGELOG.md"
```

Comment:
```suggestion
    sed -i "s/^\[unreleased\].*$/[unreleased]: https:\/\/github.com\/openfun\/richie-site-factory\/compare\/${site}-${new_version}...HEAD/gi" "${SITES_DIRECTORY}/${site}/CHANGELOG.md"
```

`/gi` instead of `/g` allows to be case insensitive and match `[Unreleased]: ...` links

Reply:
done